datasets/cord19/README.md

```diff
@@ -0,0 +1,194 @@
+---
+annotations_creators:
+- no-annotation
+language_creators:
+- expert-generated
+languages:
+- en
+licenses:
+[
+  "cc-by-nc-sa-2.0",
+  "cc-by-nc-2.0",
+  "cc-by-nd-2.0",
+  "cc-by-sa-2.0",
+  "cc-by-nc-nd-2.0",
+  "cc-by-nc-1.0",
+  "other-cc0",
+  "other-hybrid-oa",
+  "other-els-covid",
+  "other-no-cc",
+  "other-gold-oa",
+  "other-green-oa",
+  "other-bronze-oa",
+  "other-biorxiv",
+  "other-arxiv",
+  "other-medrxiv",
+  "other-unk"
+]
+multilinguality:
+- monolingual
+size_categories:
+- 100K<n<1M
+source_datasets:
+- original
+task_categories:
+- other
+task_ids:
+- other-other-knowledge-extraction
+---
+
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [https://www.semanticscholar.org/cord19](https://www.semanticscholar.org/cord19)
+- **Repository:** [https://github.com/allenai/cord19](https://github.com/allenai/cord19)
+- **Paper:** [CORD-19: The COVID-19 Open Research Dataset](https://www.aclweb.org/anthology/2020.nlpcovid19-acl.1/)
+- **Leaderboard:** [Kaggle challenge](https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge)
+
+
+### Dataset Summary
+
+CORD-19 is a corpus of academic papers about COVID-19 and related coronavirus research.  It's curated and maintained by the Semantic Scholar team at the Allen Institute for AI to support text mining and NLP research.
+
+### Supported Tasks and Leaderboards
+
+See tasks defined in the related [Kaggle challenge](https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge/tasks)
+
+### Languages
+
+The dataset is in english (en).
+
+## Dataset Structure
+
+### Data Instances
+
+TODO
```

Comment:
Could you add one example from the dataset here ?

You can get one by using `dataset["train"][0]` for example

Reply:
sure